packages/vm-config/README.md

```diff
@@ -0,0 +1,13 @@
+# VM Config
+
+Factored out of @agoric/vats, by way of @agoric/boot.
+
+This is similar to `@agoric/boot` but because that has the integration testing of bootstrap, it depends on almost everything.
+
+The configs themselves have no dependencies. Keeping them separate allows packages to depend on them without depending on the world.
```

Comment:
This seems like useful justification.

But I wonder to what extent it's true. The configs _do_ reference other packages:

```json
  "coreProposals": [
    "@agoric/vats/scripts/init-core.js",
    "@agoric/vats/scripts/init-network.js",
    {
      "module": "@agoric/inter-protocol/scripts/init-core.js",
```

Reply:
yeah… in a way it's not true. It's only true in the way that you can _get_ the package without any other dependencies.

Reply:
thinking more, this is fully true. the configs have no dependencies. the runtime that wants to build or execute those configs does.